services/bind/bind.tex

```diff
@@ -0,0 +1,11 @@
+\section{BIND}\index{bind}
+
+BIND is a common, featured DNS server. To make it more secure and less vulnerable to attacks, it is recommended to only run BIND as an authoritative nameserver and not a recursive nameserver. Below is a set of example configuration files for securely configuring BIND as an authoritative nameserver with forward and reverse records.
+
```

Comment:
Unfortunately, it isn't consistent. The main places are '/etc/bind/', '/etc/named/', and '/etc/named.conf' and '/var/named/'. I'll make a note and update the pull request.